src/Manager/initEnvironments.ts

```diff
@@ -5,128 +5,142 @@
  */
 
 import { net } from '@electron/remote';
+import {
+    AppThunk,
+    logger,
+    usageData,
+} from '@nordicsemiconductor/pc-nrfconnect-shared';
 import { execSync } from 'child_process';
 import fs from 'fs';
 import fse from 'fs-extra';
 import path from 'path';
-import { logger, usageData } from 'pc-nrfconnect-shared';
 
 import {
     persistedInstallDir as installDir,
+    persistedInstallDir,
```

Comment:
The line above already imports. 
I'm for keeping your line but the references to it below have to be renamed.